### DISCUSSION TITLE
Multiple Pfsense firewalls?

### DISCUSSION THREAD

**Discussion by tbaror** [pfelk/pfelk#342] [Q&A] [answered]
Hello ,
I have several pfsense fw and i would like to get feeds from all of the , 
 i tried to add additional pfsense send logs to current server , i don't see it in index , or logs of logstash
any idea ?
Pleaes advice
Thanks

**Comment by a3ilson** [comment, marked as answer]
How are you sending them? Same port or different ports?

I would recommend sending them all to the same port (e.g. 5140) and then you should be able to delineate the multiple firewalls by their host.name field.
